### PR TITLE
Created docs for MeshPhysicalMaterial and ShadowMaterial

### PR DESCRIPTION
**Pull request by looeee**

### CODE REVIEW

File: docs/api/materials/MeshPhysicalMaterial.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		<div>
+		Degree of reflectivity, from *0.0* to *1.0*. Default is *0.5*.<br />
+		</div>
+
+		<h3>[property:Float clearCoat]</h3>
+		<div>
+		ClearCoat level. Default is *0.0*.
+		</div>
+
+		<h3>[property:Float metalness]</h3>
```
Comment (WestLangley) [inline docs/api/materials/MeshPhysicalMaterial.html]: Should be `clearCoatRoughess`. Range is [ 0, 1 ].

File: docs/api/materials/MeshPhysicalMaterial.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		</div>
+
+
+		<h2>Properties</h2>
+		<div>See the base [page:Material] and [page:MeshStandardMaterial] classes for common properties.</div>
+
+		<h3>[property:Float reflectivity]</h3>
+		<div>
+		Degree of reflectivity, from *0.0* to *1.0*. Default is *0.5*.<br />
+		</div>
```
Comment (WestLangley) [inline docs/api/materials/MeshPhysicalMaterial.html]: `reflectivity` models the reflectivity of non-metallic materials. Range is [ 0, 1 ]. It has no effect when `metalness` is 1.

**Comment by WestLangley** [comment]
@looeee Thank you for this -- and all your contributions to the documentation.

**Comment by looeee** [comment]
@WestLangley no problem, and thanks for the feedback :)

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**